staging/nvidia/charts/nvidia-driver/templates/configmap.yaml

```diff
@@ -0,0 +1,274 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "nvidia-driver.fullname" . }}-centos7-vault-repo
+  namespace: kube-system
+data:
```

Comment:
Will this work when air-gapped?

Reply:
No, but neither will the driver install as it is now. I have an idea, but I'd like to do it as a next step.